charts/simple-app/templates/tests/test-connection.yaml

```diff
@@ -8,6 +8,11 @@ metadata:
     "helm.sh/hook": test
 spec:
   restartPolicy: Never
+  { { - if or .Values.volumes .Values.volumesString } }
```

Comment:
should I be reusing `.Values.volumes` or use `.Values.tests.connection.volumes `?